src/org/exist/Database.java

```diff
@@ -112,62 +113,64 @@
      *     database.release(broker);
      * }
      */
-    public DBBroker get(Optional<Subject> subject) throws EXistException;
+    DBBroker get(Optional<Subject> subject) throws EXistException;
 
-    public DBBroker getActiveBroker(); // throws EXistException;
+    DBBroker getActiveBroker(); // throws EXistException;
 
     /**
      * Returns the number of brokers currently serving requests for the database
      * instance.
-     * 
+     *
      * @return The brokers count
      */
-    public int countActiveBrokers();
+    int countActiveBrokers();
 
     /**
-     * 
+     *
      * @return Debuggee
      */
-    public Debuggee getDebuggee();
+    Debuggee getDebuggee();
 
-    public PerformanceStats getPerformanceStats();
+    PerformanceStats getPerformanceStats();
 
     // old configuration
-    public Configuration getConfiguration();
+    Configuration getConfiguration();
 
-    public NodeIdFactory getNodeFactory();
+    NodeIdFactory getNodeFactory();
 
-    public Path getStoragePlace();
+    Path getStoragePlace();
 
-    public CollectionConfigurationManager getConfigurationManager();
+    CollectionConfigurationManager getConfigurationManager();
 
     /**
      * Master document triggers.
      */
-    public Collection<TriggerProxy<? extends DocumentTrigger>> getDocumentTriggers();
-
-    // public DocumentTrigger getDocumentTrigger();
+    Collection<TriggerProxy<? extends DocumentTrigger>> getDocumentTriggers();
 
     /**
      * Master Collection triggers.
      */
-    public Collection<TriggerProxy<? extends CollectionTrigger>> getCollectionTriggers();
+    Collection<TriggerProxy<? extends CollectionTrigger>> getCollectionTriggers();
+
+    void registerDocumentTrigger(Class<? extends DocumentTrigger> clazz);
+
+    void registerCollectionTrigger(Class<? extends CollectionTrigger> clazz);
 
-    // public CollectionTrigger getCollectionTrigger();
+    ProcessMonitor getProcessMonitor();
 
-    public void registerDocumentTrigger(Class<? extends DocumentTrigger> clazz);
+    boolean isReadOnly();
 
-    public void registerCollectionTrigger(Class<? extends CollectionTrigger> clazz);
+    NotificationService getNotificationService();
 
-    public ProcessMonitor getProcessMonitor();
+    PluginsManager getPluginsManager();
 
-    public boolean isReadOnly();
+    SymbolTable getSymbols();
 
-    public NotificationService getNotificationService();
+    MetaStorage getMetaStorage();
 
-    public PluginsManager getPluginsManager();
+    int getPageSize();
 
-    public SymbolTable getSymbols();
+    boolean isTransactional();
```

Comment:
We should not have `isTranscational()`, it is misleading. This PR - https://github.com/eXist-db/exist/pull/989 removes that.